backend/src/credentials/listOfRequests.ts

```diff
@@ -1,27 +1,32 @@
 import * as Kilt from '@kiltprotocol/sdk-js'
 
+import { CTYPE_HASH, REQUIRED_PROPERTIES, TRUSTED_ATTESTERS } from '../config'
+
+// Here you can set which type of credential (cType) your dApp will request users to login.
+// You can change it by importing a different one.
+// The default is the Email CType by SocialKYC and SocialKYC as the Issuer
 // Establish which cTypes our dApp accepts and which attesters we trust:
 
```

Comment:
I think that all this comments could go inside the variable explainer (docs) from `requestedCTypeForLogin`. Maybe a bit more structured and explaining how to change the imports.